adapters/WorldGuardAdapter/src/main/kotlin/com/caleb/typewriter/worldguard/entries/event/ExitRegionEventEntry.kt

```diff
@@ -10,7 +10,7 @@ import me.gabber235.typewriter.entry.entries.EventEntry
 import me.gabber235.typewriter.utils.Icons
 import java.util.*
 
-@Entry("on_exit_region", "When a player exits a WorldGuard region", Colors.YELLOW, Icons.SQUARE_XMARK)
+@Entry("on_exit_wg_region", "When a player exits a WorldGuard region", Colors.YELLOW, Icons.SQUARE_XMARK)
```

Comment:
Though I do agree with this change, don't include it in this pr. Such a changes requires me to create a migration flow as users are possibly already using the this entry and making this update will break their pages. 

Maybe create a separate pr for it.